services/state/token/token.proto

```diff
@@ -38,9 +38,9 @@ option java_multiple_files = true;
 
 message Token {
     /**
-     * The unique entity number of this token.
+     * The unique entity id of this token.
      */
-    int64 token_number = 1;
+    TokenID token_id = 1;
```

Comment:
Nit: We've been a bit inconsistent with these.
Some services have just `id`, others have `thing_id`.
I would argue that typing `Token.tokenId()` is redundant and these should be just `id`/`Token.id()`.
Might be wise to bring up in the modularization call so it can be consistent.

Reply:
Yes, agreed. we see id in topic.proto, but file_id in file.proto, token_id in token.proto, etc. There are so many related  code changes for just this small name change.